.github/workflows/node-e2e-tests.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Node e2e tests"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  node-e2e-tests:
+    # only run this job if the base branch is dp/testing-groups
+    if: ${{github.ref_name == '728/merge'}}
```

Comment:
We are having to keep this here because we will need to merge this PR in before being fully done with this whole setup, as things stand. Please see: https://github.com/FuelLabs/fuels-ts/pull/728#issuecomment-1446348803